### ISSUE TITLE
关于pytorch模型在ncnn中的推理异常问题

### ISSUE THREAD

**Issue by tomingliu** [Tencent/ncnn#973]
在pytorch推理中，数据是这样的输入的

    def get_data(self,image_path):
        img = cv2.imread(image_path)
        return np.transpose(np.array(img, dtype = np.float32), (2, 0, 1)), image_path

    def inference(**kwargs):
        data,_ = data_handle.get_data(imgdir)
        data = data[np.newaxis,:]
        data = torch.FloatTensor(data)
        with torch.no_grad():
            if opt.use_gpu:
                data =  data.cuda()
            outputs = model(data)

对应ncnn的推理实现

void test(Mat &face_pic)
{
    static const float normalize[] = {
        1.f/255, 1.f/255, 1.f/255
    };
    float sum = 0.0; int i;

    ncnn::Extractor ex = Knet.create_extractor();

    ncnn::Mat in = ncnn::Mat::from_pixels(
            face_pic.data,
            ncnn::Mat::PIXEL_BGR,
            face_pic.cols, face_pic.rows
    );

    in.substract_mean_normalize(NULL, normalize);// pytorch的to_tensor做了除255的操作

    ex.set_num_threads(4);
    ex.set_light_mode(true);
    ex.input("data", in);
    ex.extract("fc5", out);

两者输出的结果不一致，大伽帮忙看看，对应的NCNN的输入部分是不是预处理的不对？

**Comment by tomingliu** [comment]
模型转换问题，已经解决

**Comment by xfarxod** [comment]
Hi @tomingliu . I have the same problem. How you resolve this issue.? How you convert your pytorch model to ncnn?

**Comment by wait1988** [comment]
+1

**Comment by liguiyuan** [comment]
> 模型转换问题，已经解决

你好，请问一下你模型转换遇到的是什么问题呢？
